doozer/doozerlib/telemetry.py

```diff
@@ -0,0 +1,94 @@
+import os
```

Comment:
`telemetry.py` is basically copied across doozer/pyartcd, with some minor differences. Can this module be a common one, and placed in `art-common` when ready?

Reply:
Yes, I am planing to do that when art-common is ready.